modules/miraheze.py

```diff
@@ -36,12 +37,14 @@ def miraheze(bot, trigger):
 
 
 @commands('gethelp')
+@rule('[i-iI-I] need help|[c-cC-C]an someone help me|[i-iI-I] can(t|'t) login).*')
 @example('.gethelp I cannot access https://meta.miraheze.org')
 def miraheze_gethelp(bot, trigger):
     """Reply to help requests."""
     if trigger.sender == '#miraheze':
-        bot.say(trigger.nick + ', needs help. Pinging Reception123, Zppix, '
-                'PuppyKun, Voidwalker, RhinosF1.')
+        bot.reply('I am sure that Reception123, Zppix, PuppyKun, Voidwalker, RhinosF1, or Examknow would be happy to help you.')                                                    
```

Comment:
That’s god awful. Use something like “pinging Reception123, ... who might be able to help you.”